src/Application/GameConfig.h

```diff
@@ -157,6 +157,10 @@ namespace Application {
                                     "Treat clubs as maces. "
                                     "In vanilla clubs are using a separate hidden skill and can be equipped without learning the mace skill."};
 
+            Bool FixWaterWalkManaDrain = {this, "fix_water_walk_mana_drain", true,
+                                          "Fix water walk mana drain interval. "
```

Comment:
Let's phrase it a bit differently so that it's more obvious what's going on.

"Change water walk mana drain interval to 20 minutes. Spell description in vanilla says that mana is drained every 20 minutes, but in reality, it's every 5 minutes."